### ISSUE TITLE
[Bug Report] All newly added scenes have the exact same thumbnail and preview video

### ISSUE THREAD

**Issue by Erovall** [stashapp/stash#3075]
**Describe the bug**
All newly added scenes have an identical thumbnail which is taken from one of the scenes. I have performed a Clean already. I've also tried using one of my backed up databases and rescan after that, the bug persists. I've also tried the `Generate` task with `overwrite` enabled and still no help. In fact, in this case the generated preview video is also the same across all the newly added scenes!

**To Reproduce**
The bug is reproducible on my database. Just scan any new scene.

**Stash Version: (from Settings -> About):**
v0.17.2

**Additional context**
Looking inside the `generated\screenshots` folder I see files named exactly `.jpg` and `.mp4`. These are precisely the image and preview video being played on all the newly added scenes. For previously added scenes I see multiple files in this folder with unique filenames.

Before this issue started, I _may_ have closed my browser (Chrome on Desktop) during a Scan task. But I'm not sure if I closed it during the Scan, but it's possible.

**Update**
Ok I copied the stash exe and my sql database to a new folder and ran Stash again from there. Then I re-scanned the scenes folder and now all the scenes have the correct thumbnails. So, I guess something is messed up in my old stash working directory folder.

**Comment by MrX292** [comment]
can you check if the new filenames in `generated\screenshots` are longer or shorter then the old ones?
do you also know what `Generated file naming hash` was set to and if it is different

**Comment by Erovall** [comment]
New scenes are not generating any new filenames in `generated\screenshots`. When I scan, it only gets 2 more files (exactly 2. Not 2 for each new scene) and they are named exactly `.jpg` and `.mp4` i.e. only the extension, no name. Older scenes have their filenames properly.

No idea what `Generated file naming hash` is or what it was set to.

I have an update: I deleted the entire generated folder and re-scanned the scenes. This still leads to the same bug as in the opening post. I deleted the `config.yml` as well, and then restarted Stash and scanned again, and now everything is fixed. So something must be wrong in `config.yml`. Sorry that I deleted the file and don't have it anymore - maybe it had some clue. But hopefully this helps?
